web/source/osk/oskManager.ts

```diff
@@ -1022,7 +1050,8 @@ namespace com.keyman.osk {
         }
       }
 
-      return height;
+      let bannerHeight = this.banner.height;
+      return height + bannerHeight;
```

Comment:
I would strongly prefer a minor rework of this method - perhaps a `getKeyboardHeight`, `getBannerHeight`, and a combined `getHeight`.  `VisualKeyboard` shouldn't need to worry about influence from the banner if we can avoid it.